libs/go.json

```diff
@@ -218,5 +218,8 @@
   {"id": "go.google-email-audit-api", "imports": ["github.com/ngs/go-google-email-audit-api"], "name": "Google Email Audit API", "repo": "ngs/go-google-email-audit-api", "tags": ["google-email-audit"], "tech": ["api"], "status": "model-ready"},
   {"id": "go.golf", "imports": ["github.com/dinever/golf"], "name": "golf", "repo": "dinever/golf", "tags": ["framework"], "tech": ["go-web"], "status": "model-ready"},
   {"id": "go.slack-api", "imports": ["github.com/nlopes/slack"], "name": "Slack API", "repo": "nlopes/slack", "tags": ["slack"], "tech": ["chat-ux"], "status": "model-ready"},
-  {"id": "go.golarm", "imports": ["github.com/msempere/golarm"], "name": "golarm", "repo": "msempere/golarm", "tags": ["alarms"], "tech": ["diagnostics"], "status": "model-ready"}
+  {"id": "go.golarm", "imports": ["github.com/msempere/golarm"], "name": "golarm", "repo": "msempere/golarm", "tags": ["alarms"], "tech": ["diagnostics"], "status": "model-ready"},
+  {"id":"go.logrus","imports":["github.com/sirupsen/logrus"],"name":"logrus","repo":"https://github.com/Sirupsen/Logrus","tags":["logging"],"tech":["diagnostics"],"status":"awaiting-model"},
```

Comment:
I am curious about these fully qualified imports with github.com. could you confirm with @yaronskaya that we support those well?

Reply:
It's ok, we extract such imports.